articles/quickstarts/install-command-line.md

```diff
@@ -18,7 +18,11 @@ Q# programs can be executed on their own, without a driver in a host language li
 
 ## Installation
 
-While you can build Q# command line applications in any IDE, we recommend using Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+While you can build Q# command line applications in any IDE, we recommend using Visual Studio Codespaces, Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+
+To configure Visual Studio Codespaces:
+1. Create an Azure account [here](https://azure.microsoft.com/en-us/free/).
+2. Create a Codespaces environment. Please follow the quickstart guide [here](https://docs.microsoft.com/visualstudio/online/quickstarts/browser). When creating the Codespace, make sure to enter microsoft/Quantum in the "Git Repository" field.
```

Comment:
This makes sense for creating a Codespace from the samples repo, but do we want to provide any steps that users can follow to use VS Codespaces with their own projects?

Reply:
Yes that is a great point... We should probably make this more general.
Would it be an idea to create a template repo (like https://github.com/cgranade/quantum-research-template) and link that here?

Reply:
I think that could make a lot of sense, yeah. Probably beyond the scope of this PR, but something we can do fairly quickly afterwards.

Reply:
Sounds good, I'll add a note here to specify that any repo could be used, but we recommend `microsoft/Quantum` by default.